src/main.cpp

```diff
@@ -92,39 +95,44 @@ void loop() {
      * network (WiFiUdp will print an warning message in those cases).
      */
     if (puara.IP1_ready()) { // set namespace and send OSC message for address 1
-        OSCMessage msg1(("/" + puara.get_dmi_name() + "/X").c_str());
-        OSCMessage msg2(("/" + puara.get_dmi_name() + "/Y").c_str());
-        OSCMessage msg3(("/" + puara.get_dmi_name() + "/Z").c_str());
-        msg1.add(event.orientation.x);
-        msg2.add(event.orientation.y);
-        msg3.add(event.orientation.z);
-        Udp.beginPacket(puara.getIP1().c_str(), puara.getPORT1());
-        msg1.send(Udp);
+        
+        // Create bundle to allow sending multiple OSC messages at once
+        OSCBundle bundle;
+
+        String baseMessage = ("/" + puara.dmi_name()).c_str();
```

Comment:
It is probably better to use `std::string` and have this outside the loop. We just need to call it once. Move it out of the loop will work well together with my other comment below